site.yml.sample

```diff
@@ -75,6 +75,7 @@
   roles:
     - ceph-defaults
     - ceph-validate
+    - ceph-infra
```

Comment:
would be nice to have tags for each subcomponent such as: 'configure_ntp', 'configure_firewall' so one can skip ntp if they want to.